.github/workflows/test_definitions.yml

```diff
@@ -6,53 +6,56 @@ on:
   workflow_call:
     # Inputs and secrets documented in the caller workflows
     inputs:
-      clang-tidy:
+      ros-ci:
         required: true
         type: boolean
-      custom-interfaces:
+      clang-tidy:
         required: true
         type: boolean
-      disable-vcs:
-        required: true
-        type: string
       repository:
         required: true
         type: string
       rebuild-docs:
         required: true
         type: boolean
+      markdown-ci:
+        required: false
```

Comment:
My perspective is to make everything required so that all arguments can be found in tests.yml and all tests.yml should look the same. What do you think?

Reply:
But I also understand it's a pain to update every test.yml file every time an input is added/removed. We could mitigate this by pointing the tests.yml's only to dev branches, not main

Reply:
I like the idea of a separate repo and just letting dependabot handle it.

Reply:
Me too. In that case I think it's fine to leave the PR as is. Can you write an issue for this?

Reply:
Actually we don't even need to put the workflow in its own repo, we can reference a release instead of a branch. Dependabot supports this: https://github.blog/changelog/2023-03-13-dependabot-updates-support-reusable-workflows-for-github-actions/